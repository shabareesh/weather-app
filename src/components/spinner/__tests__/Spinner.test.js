import React from 'react';
import { render } from "@testing-library/react";
import Spinner from "../Spinner";

describe('Spinner', () => {
  test('should match snapshot', () => {
    const { baseElement } = render(<Spinner />);
    expect(baseElement).toMatchSnapshot();
  });
});
