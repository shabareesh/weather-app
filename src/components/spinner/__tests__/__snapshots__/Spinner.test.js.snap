// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match snapshot 1`] = `
<body>
  <div>
    <div
      class="spinner"
    />
  </div>
</body>
`;
